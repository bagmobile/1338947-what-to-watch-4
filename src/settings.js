export const EMPTY_FIELD_VALUE = `Данные отсутствуют`;
export const ANONYMOUS_AUTHOR = `Анонимный пользователь`;
