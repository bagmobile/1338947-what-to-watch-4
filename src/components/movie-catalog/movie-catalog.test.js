import React from "react";
import renderer from "react-test-renderer";
import MovieCatalog from "./movie-catalog";
import {genres} from "../../settings.js";

describe(`MovieCatalog component`, () => {

  it(`Render`, () => {
    const tree = renderer
      .create(<MovieCatalog
        genres={genres}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

});
