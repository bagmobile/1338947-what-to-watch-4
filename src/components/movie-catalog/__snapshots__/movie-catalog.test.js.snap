// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCatalog component Render 1`] = `
Array [
  <h2
    className="catalog__title visually-hidden"
  >
    Catalog
  </h2>,
  <ul
    className="catalog__genres-list"
  >
    <li
      className="catalog__genres-item catalog__genres-item--active"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        All genres
      </a>
    </li>
    <li
      className="catalog__genres-item false"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Crime
      </a>
    </li>
    <li
      className="catalog__genres-item false"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Documentary
      </a>
    </li>
    <li
      className="catalog__genres-item false"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Comedies
      </a>
    </li>
    <li
      className="catalog__genres-item false"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Sci-Fi
      </a>
    </li>
    <li
      className="catalog__genres-item false"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Romance
      </a>
    </li>
    <li
      className="catalog__genres-item false"
    >
      <a
        className="catalog__genres-link"
        href="#"
      >
        Dramas
      </a>
    </li>
  </ul>,
]
`;
